corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -31,8 +31,11 @@ hqDefine("cloudcare/js/formplayer/menus/views", [
     L
 ) {
     const MenuView = Marionette.View.extend({
+        isGrid: function () {
+            return this.model.collection.layoutStyle === constants.LayoutStyles.GRID;
+        },
```

Comment:
Could this just be a boolean rather than a function?
```javascript
isGrid: this.model.collection.layoutStyle === constants.LayoutStyles.GRID,
```

Reply:
no, it needs to be in a function for `this` to exist